### PR TITLE
operators project-quay (3.7.0-rc.2)

### PR DESCRIPTION
**Pull request by quay-devel**
Add v3.7.0-rc.2 to community operators

### CODE REVIEW

**Comment by HammerMeetNail** [comment]
@J0zi @mvalarh Can you help us understand why the `Operator test / PR-traffic-light (pull_request_target)` test is failing? This is our first z-stream bump and we're having some trouble understanding the error message:

```
Bundle channels information not provided, inferring from parent package directory
Error: error interpreting channels, please manually input channels instead
```

How can I specify the channel for the old version we are replacing?

File: operators/project-quay/3.7.0-rc.2/quay-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+    - email: quay-sig@googlegroups.com
+      name: Project Quay Contributors
+  provider:
+    name: Red Hat
+  selector:
+    matchLabels:
+      alm-owner-quay-operator: quay-operator
+      operated-by: quay-operator
+  version: 3.7.0-rc.2
+  replaces: 3.7.0-rc.1
```
Comment (mvalarh) [inline operators/project-quay/3.7.0-rc.2/quay-operator.clusterserviceversion.yaml:285]: Value for replaces should be `quay-operator.v3.7.0-rc.1`

**Review by mvalarh — APPROVED** [review]

File: operators/project-quay/3.7.0-rc.2/quay-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (J0zi) [reply operators/project-quay/3.7.0-rc.2/quay-operator.clusterserviceversion.yaml:285]: @HammerMeetNail ^

Comment (HammerMeetNail) [reply operators/project-quay/3.7.0-rc.2/quay-operator.clusterserviceversion.yaml]: Thanks! I've pushed the change and that test is passing now. Fingers crossed the rest of the tests pass and we an merge this in today :)

**Review by HammerMeetNail — APPROVED** [review]
LGTM 👍

**Comment by HammerMeetNail** [comment]
@J0zi @mvalarh Can either of you add the `authorized-changes` label? I'll get in a separate PR to update our list of reviewers.

**Comment by J0zi** [comment]
@HammerMeetNail we encourage you you to edit list of reviewers anyway. It will merge your PR automatically next time.

**Pull request merged**